models/eos/interfaces/deleted_example_01.txt

```diff
@@ -0,0 +1,34 @@
+
+# Using deleted
+
+# Before state:
+# -------------
+#
+# veos#show running-config | section interface
+# interface Ethernet1
+#    description "Interface 1"
+# !
+# interface Ethernet2
+# !
+# interface Loopback1
+# !
+# interface Management1
+#    ip address dhcp
+# !
+
+- name: Configure interfaces
+  eos_interfaces:
+    operation: deleted
+
+# After state:
+# ------------
+#
+# veos#show running-config | section interface
+# interface Ethernet1
+#    description "Interface 1"
```

Comment:
Since the operation is `deleted`, and the and the scope was not limited by a list of interfaces when using the module, and `description` is a attribute the interfaces modules is responsible for, this interfaces should have had it's description removed.